Packs/Symantec_Advanced_Threat_Protection/Integrations/integration-Symantec_Advanced_Threat_Protection.yml

```diff
@@ -28,6 +28,45 @@ configuration:
   defaultvalue: ""
   type: 8
   required: false
+- display: Incident data source
+  name: fetch_incidents_type
+  defaultvalue: incidents
+  type: 15
+  required: false
+  options:
+  - incidents
+  - events
+  - incidentevents
+- display: Query string for fetch incidents
+  name: fetch_incidents_query
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Maximum number of events per fetch.
+  name: fetch_incidents_limit
+  defaultvalue: "200"
+  type: 0
+  required: false
+- display: Fetch incidents
+  defaultvalue: ""
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  defaultvalue: ""
+  name: incidentType
+  type: 13
+  required: false
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: first_fetch_time
+  defaultvalue: 1 day
+  type: 0
+  required: false
+- display: Fetch incidents over the server limit
+  name: exceed_server_limit_fetching
+  defaultvalue: "false"
+  type: 8
+  required: false
```

Comment:
are we sure we wnat this parameter?

Reply:
Yes. In the use case of the customer who requested the feature, they get thousands of events within a minute even setting a query.
On the other hand, they can drop most of incidents in pre-processing rules which is more capable than the query SATP accepts.
So, they need the fetch incidents capable to get a number of events within a fetch cycle.

Reply:
Instead, I've added HARD_LIMIT (10000) to avoid too much memory consumption.

Reply:
let's remove this, fetching over 100 incidetns in a minute that triggers a playbook is not a good practice.

Reply:
The customer have been using a pre-processing rule with some conditions to drop incidents. A playbook is not triggered so much within a minute. In reality, they get 10 or less incidents in a minute though they get over 100 incidents from the ATP.

Reply:
Query string doesn't work well to filter incidents they want, whereas pre-processing rules can. So they need to retrieve incidents once into XSOAR to filter in the pre-processing rules.

Reply:
I see. it is great that THIS customer is using pre-processing rules to filter incidents, so you can rm the param `fetch_incidents_query` but nothing ensures us that future customers will. So my request to harden the fetch limit still stands.